impl/src/main/java/org/glassfish/json/JsonReaderImpl.java

```diff
@@ -36,23 +36,28 @@
  * @author Jitendra Kotamraju
  */
 class JsonReaderImpl implements JsonReader {
+    public static final String ALLOW_DUPLICATE_KEYS = "org.glassfish.json.JsonReaderImpl.allowDuplicateKeys";
```

Comment:
I'd change the property name to `org.glassfish.json.allowDuplicateKeys` instead and if the intention is to be available to users, consider defining this in the `api` package which is being exported in JPMS/OSGi

Reply:
I made the change that you have recommended. Please see my latest commit.